components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/util/ClientCredentialAuthConfig.java

```diff
@@ -146,6 +148,13 @@ public boolean shouldRetry() {
     public HttpUriRequest applyAuth(HttpUriRequest request, AuthConfigModel authConfigModel)
             throws FrameworkException {
 
+        if (request.getMethod().equals(Constants.GET)) {
+            INVOKE_API = Constants.LogConstants.ActionIDs.INVOKE_API_HTTP_GET;
+            REQUEST_TOKEN = Constants.LogConstants.ActionIDs.REQUEST_TOKEN_HTTP_GET;
+        } else if (request.getMethod().equals(Constants.POST)) {
+            INVOKE_API = Constants.LogConstants.ActionIDs.INVOKE_API_HTTP_POST;
+            REQUEST_TOKEN = Constants.LogConstants.ActionIDs.REQUEST_TOKEN_HTTP_POST;
+        }
```

Comment:
Even this part better resolve before the log. Because for every success operation this section doesn't have a meaning

Reply:
Fixed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/161/commits/81b4b372496a3e2c4ba0778f71588e5a3416546a